automation/taskcluster/schedule_master.py

```diff
@@ -0,0 +1,98 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file looks like a quick way of testing `.taskcluster.yml`. It's also a copy of `schedule_nightly.py`. I would:
* either remove this file from the PR
* or factorize schedule_nightly.py to support both cases.

Reply:
I'm going to remove this file from the PR, it's just for testing my changes :)